packages/lib/convertToNewDurationType.ts

```diff
@@ -0,0 +1,43 @@
+export const MINUTES_IN_HOUR = 60;
+export const MINUTES_IN_DAY = 1440;
+export const HOURS_IN_DAY = 24;
+
+export default function convertToNewDurationType(
```

Comment:
`"string" | "minutes" | "hours" | "days"` is equivalent to `string`
<img width="465" alt="Screenshot 2022-11-11 at 5 00 53 PM" src="https://user-images.githubusercontent.com/1780212/201331691-2477fa73-b79c-4729-a64a-63272fa37458.png">